packages/smart-contract/src/network/native-api.ts

```diff
@@ -42,11 +44,11 @@ export async function sendTransaction(json: any, url: string): Promise<TSendTran
     const { data, errors } = response.data;
 
     if (errors) {
-      throw Error(errors)
+      throw errors;
     }
 
-    return data;
+    return data.sendZkapp.zkapp;
```

Comment:
Is this any better way to validate before return?

Reply:
Yep, I added the validate on my new commit